$NetBSD: patch-aa,v 1.3 2016/06/18 11:30:35 mef Exp $

date: 2009-11-01 06:42:27 +0900;  author: joerg;  state: Exp;  lines: +1 -10;
Drop unneeded fragment from patch.

date: 2009-10-30 01:25:57 +0900;  author: joerg;  state: Exp;
Do not stupidly insist on GNU gettext, thanks.

--- configure.orig	2016-05-17 06:02:07.000000000 +0900
+++ configure	2016-06-18 19:29:50.000000000 +0900
@@ -18179,23 +18179,7 @@ $as_echo "#define HAVE_CFLOCALECOPYCURRE
 
     if test "$USE_NLS" = "yes"; then
     gt_use_preinstalled_gnugettext=no
-
-
-        if test $gt_api_version -ge 3; then
-          gt_revision_test_code='
-#ifndef __GNU_GETTEXT_SUPPORTED_REVISION
-#define __GNU_GETTEXT_SUPPORTED_REVISION(major) ((major) == 0 ? 0 : -1)
-#endif
-typedef int array [2 * (__GNU_GETTEXT_SUPPORTED_REVISION(0) >= 1) - 1];
-'
-        else
           gt_revision_test_code=
-        fi
-        if test $gt_api_version -ge 2; then
-          gt_expression_test_code=' + * ngettext ("", "", 0)'
-        else
-          gt_expression_test_code=
-        fi
 
         { $as_echo "$as_me:${as_lineno-$LINENO}: checking for GNU gettext in libc" >&5
 $as_echo_n "checking for GNU gettext in libc... " >&6; }
@@ -18883,15 +18867,11 @@ else
 $gt_revision_test_code
 extern int _nl_msg_cat_cntr;
 extern
-#ifdef __cplusplus
-"C"
-#endif
-const char *_nl_expand_alias (const char *);
 int
 main ()
 {
 bindtextdomain ("", "");
-return * gettext ("")$gt_expression_test_code + _nl_msg_cat_cntr + *_nl_expand_alias ("")
+return * gettext ("")$gt_expression_test_code + _nl_msg_cat_cntr
   ;
   return 0;
 }
@@ -18911,15 +18891,11 @@ rm -f core conftest.err conftest.$ac_obj
 $gt_revision_test_code
 extern int _nl_msg_cat_cntr;
 extern
-#ifdef __cplusplus
-"C"
-#endif
-const char *_nl_expand_alias (const char *);
 int
 main ()
 {
 bindtextdomain ("", "");
-return * gettext ("")$gt_expression_test_code + _nl_msg_cat_cntr + *_nl_expand_alias ("")
+return * gettext ("")$gt_expression_test_code + _nl_msg_cat_cntr
   ;
   return 0;
 }
